src/Microsoft.Health.Dicom.Core/Configs/FeatureConfiguration.cs

```diff
@@ -18,8 +18,8 @@ public class FeatureConfiguration
         public bool EnableFullDicomItemValidation { get; set; }
 
         /// <summary>
-        /// Enables CustomTagQueryTags feature.
+        /// Enables ExtendedQueryTags feature.
         /// </summary>
-        public bool EnableCustomQueryTags { get; set; }
```

Comment:
please double check if appsettings is also updated.
appsettings.json and appsettings.development.json